docs/topics/using/edgectl/service-preview-reference.md

```diff
@@ -8,7 +8,7 @@ See [Service Preview Quick Start](../service-preview-install) for detailed insta
 
 The Traffic Manager is the central point of communication between Traffic Agents in the cluster and Edge Control Daemons on developer workstations.
 
-The following YAML is the basic Traffic Manager installation manifests that is available for download at https://www.getambassador.io/yaml/traffic-manager.yaml.
+The following YAML is the basic Traffic Manager installation manifests that is available for download at https://getambassador.io/yaml/traffic-manager.yaml.
```

Comment:
Is there a reason you deleted the `www`? (this forces a 301, so want to make sure we're confident that all clients can handle this properly)